Packs/GitLab/Integrations/GitLabEventCollector/GitLabEventCollector.py

```diff
@@ -31,16 +31,26 @@ def prepare_time_for_next(time_var: str) -> str:
         return temp_time.isoformat()
 
     @staticmethod
-    def get_last_run(events: list) -> dict:
+    def get_last_run(events: list, last_run: dict) -> dict:  # type: ignore
         groups = [event for event in events if event.get('entity_type') == 'Group']
         groups.sort(key=lambda k: k.get('id'))
         projects = [event for event in events if event.get('entity_type') == 'Project']
         projects.sort(key=lambda k: k.get('id'))
```

Comment:
Extract a utility method that does it generically, with the params `events`, and `entity_type`.